draft-trammell-rains-protocol.md

```diff
@@ -205,23 +225,23 @@ query services on their behalf.
 The RAINS Protocol is based on an information model built around two kinds of
 information: Assertions and Queries. An Assertion contains some information
 about a name or address, and a Query contains a request for information about a
-name of address. The information model in this section omits information
+name or address. The information model in this section omits information
 elements required by the resolution mechanism itself; these are defined in more
 detail in {{cbor}} and {{protocol-def}}.
 
 ## Assertion
 
-An Assertion is a signed statement about a mapping from a subject name to an
-object value, and consists of the following elements:
+An Assertion is a signed statement about a mapping from a subject name to one or
+several object values of the same type, and consists of the following elements:
 
 - Context: name of the context in which the assertion is valid;
   see {{context-in-assertions}} below.
-- Subject: name about which the assertion is made. 
+- Subject: name about which the assertion is made. It MAY contain dots ('.').
```

Comment:
I'd leave "may contain dots" out -- this level of detail is not necessarily appropriate for a terminology section.

Reply:
It is not in the terminology section but in the Information Model section. Should I still remove it? It is implicitly mentioned in the 'Context in Assertions' section where based on the example one could figure out that it is the case.

Reply:
It is already mentioned in the cbor model, so yes, I will remove it from the 'Context in Assertions' section .